### PR TITLE
pAI Update Assembled: Redux

### PR DESCRIPTION
**Pull request by Geevies**
* pAI are slower.
* pAI can doorjack airlocks with bolted doors, and they will raise as long as the bolt wire isn't cut.
   [The AI only has a 60% chance to see the hack now.]
* pAI can no longer strip people.

### CODE REVIEW

**Comment by Karolis2011** [comment]
Aka known as discount #6814

**Comment by Geevies** [comment]
This not only tackles something entirely different than what you have, but it's also actually finished.

File: code/modules/mob/living/silicon/pai/software_modules.dm [hunk 1]
```diff
@@ -354,11 +354,11 @@
 /mob/living/silicon/pai/proc/hackloop()
 	var/turf/T = get_turf_or_move(src.loc)
 	for(var/mob/living/silicon/ai/AI in player_list)
-		if(T.loc)
+		if(T.loc && prob(60)) //60% chance that AI notices hack
```
Comment (Arrow768) [inline code/modules/mob/living/silicon/pai/software_modules.dm]: As discussed on discord: Remove the chance. 
AI should always get a alert if someone tries to hack a airlock with a pAI.

File: code/modules/mob/living/silicon/pai/software_modules.dm [hunk 2]
```diff
@@ -375,6 +375,7 @@
 			return
 		if(hackprogress >= 1000)
 			hackprogress = 0
+			D.unlock() //unbolts door as long as bolt wires aren't cut
```
Comment (Arrow768) [inline code/modules/mob/living/silicon/pai/software_modules.dm]: I am unsure about unbolting, but I guess we can see how it goes.

File: code/modules/mob/living/silicon/pai/software_modules.dm [hunk 3]
```diff
@@ -454,13 +455,11 @@
...
 			user.add_language(LANGUAGE_SIIK_MAAS)
 			user.add_language(LANGUAGE_SKRELLIAN)
-//			user.add_language("Vaurcese")
 			user.add_language("Rootsong")
 		else
 			user.remove_language(LANGUAGE_UNATHI)
 			user.remove_language(LANGUAGE_SIIK_MAAS)
 			user.remove_language(LANGUAGE_SKRELLIAN)
-			//user.add_language(LANGUAGE_VAURCA) //I can buy an AI core and its cyborgs having access to the local hivenet for security reasons, but a pAI?
 			user.add_language("Rootsong")
```
Comment (Arrow768) [inline code/modules/mob/living/silicon/pai/software_modules.dm]:
```suggestion
			user.add_language(LANGUAGE_ROOTSONG)
```

File: code/modules/mob/living/silicon/pai/software_modules.dm [hunk 4]
```diff
@@ -454,13 +455,11 @@
 			user.add_language(LANGUAGE_UNATHI)
 			user.add_language(LANGUAGE_SIIK_MAAS)
 			user.add_language(LANGUAGE_SKRELLIAN)
-//			user.add_language("Vaurcese")
 			user.add_language("Rootsong")
```
Comment (Arrow768) [inline code/modules/mob/living/silicon/pai/software_modules.dm]:
```suggestion
			user.add_language(LANGUAGE_ROOTSONG)
```

File: code/modules/mob/living/silicon/pai/software_modules.dm (hunk 1 above)
Comment (Geevies) [inline code/modules/mob/living/silicon/pai/software_modules.dm]:
```suggestion
		if(T.loc)
```

**Pull request merged**